src/_locales/sv/messages.json

```
@@ -54,5 +54,38 @@
   },
   "btnFullScreen": {
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
```

```suggestion
    "message": "Kortkommandon"
```

I believe the norm is to use "Kortkommandon", which directly translates to "short commands".